src/Resources/config/listener.yml

```diff
@@ -33,14 +41,10 @@ services:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 30 }
             - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
 
-    contao.listener.exception:
-        class: Contao\CoreBundle\EventListener\ExceptionListener
-        arguments:
-            - "%contao.pretty_error_screens%"
-            - "@templating"
-            - "@contao.adapter.config"
+    contao.listener.exception_converter:
+        class: Contao\CoreBundle\EventListener\ExceptionConverterListener
         tags:
-            - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
+            - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: 64 }
```

Comment:
I don't think so, as we are running the same chain again (and not pass our converted exception to the next listener).